racetime/views/user.py

```diff
@@ -134,6 +134,37 @@ def get(self, request, *args, **kwargs):
         return resp
 
 
+class UserProfileData(ViewProfile):
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        user = self.object.api_dict_summary()
+        entrances = self.get_entrances()
+        paginator = Paginator(entrances, 10)
+        resp = http.JsonResponse({
+            'id': user['id'],
+            'full_name': user['full_name'],
+            'name': user['name'],
+            'discriminator': user['discriminator'],
+            'url': user['url'],
+            'avatar': user['avatar'],
+            'pronouns': user['pronouns'],
+            'flair': user['flair'],
+            'twitch_name': user['twitch_name'],
+            'twitch_channel': user['twitch_channel'],
```

Comment:
Rather than putting all these keys in manually, you can just splat the `user` dict in with one line:
```
resp = http.JsonResponse({
    **user,
    'join_date': str(self.object.date_joined),
    'stats': {
        # ...
    }.
})
```

Reply:
I left it the way I did because I'm generally not a fan on deleting out of a dictionary if it ends up being implemented in a different way later. I had to specifically exclude can_moderate as under a user context it would not be relevant.